pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -96,6 +96,13 @@
       <Versions>10.10;10.11;10.12;10.13;10.14;10.15</Versions>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="osx">
+        <Parent>unix</Parent>
+       <Architectures>x64;arm64</Architectures>
+       <Versions>11;11.0</Versions>
```

Comment:
Because you've made osx11 a separate entry it doesn't treat 11 as compatible with 10.  Is that what we want?

Reply:
A similar discussion occurred with Tizen here: https://github.com/dotnet/runtime/pull/38832

Reply:
can we make the x64 compatible while adding arm64 only for 11? 
Or should we add arm64 for all versions even if we do not have implementation? 
It seems like Tizien took the later approach, right? 

Master change is https://github.com/dotnet/runtime/pull/40644

Reply:
Third option would be to add functionality to special case as I suggested.  It's up to you to define what you think the best behavior is.  I'm not sure treating them as incompatible is correct, but I don't know enough about Apple's versioning and how their SDK's / platform API changes release-to-release.  Perhaps some folks like @bartonjs or @akoeplinger might have an opinion.

Reply:
cc: @marek-safar since he opened the original issue. AFAK, we not use same bits build on 10.x and it seems to work find with few issues. I don't mind taking look at the generation but I'm not sure how do I ensure correctness.

Reply:
RIDs are all about *what is most useful* rather than what is *correct*.  You can always have a case where things won't work, but that's not a reason to make them incompatible: folks can always include two assets to deal with an incompatibility.  If it is more useful to treat osx11 as compatible with osx10 (so that folks can reuse a single asset the majority of the time) then that is how we should represent it.

Reply:
yes osx11 should definitely be compatible with osx10.